### PR TITLE
Added minecraft font

### PR DESCRIPTION
**Pull request by Awe23123**

### CODE REVIEW

File: seedinfo/seedinfo.html [hunk 1]
```diff
@@ -176,17 +23,15 @@
...
-						</div>
+						<div>If you want to use this tool you have to allow scripts for this site, or switch to a browser that supports JavaScript.</div>
 					</div>
 				</div>
 			</noscript>
 			<div class="listelement info" id="about" style="display:inherit">
 				<img class="listimg" src="img/logo_infoi.png"></img>
 				<div class="listdesc">
 					<div class="listhead">About this tool</div>
-					<div class="listtext">
```
Comment (Awe23123) [inline seedinfo/seedinfo.html:189]: The "listtext" class is empty so I removed it

File: seedinfo/seedinfo.html [hunk 2]
```diff
@@ -251,126 +96,90 @@
...
+					<div>
 						So I generated a random seed for you as Minecraft would do too :-)<br>
 						Seed: <span class="numseed seed"></span>
 					</div>
 				</div>
 			</div>
 			<div class="listelement special" id="seed3">
 				<img class="listimg" src="img/logo_seed3.png"></img>
-				<div class="listdesc">
```
Comment (Awe23123) [inline seedinfo/seedinfo.html:262]: Removed excessive code

File: seedinfo/seedinfo.html [hunk 3]
```diff
@@ -464,42 +273,6 @@
 	<script src="js/NumHelper.js"></script>
 	<script src="js/SeedInfo.js"></script>
 	<script src="js/ShowSeedInfo.js"></script>
-	<script>
```
Comment (Awe23123) [inline seedinfo/seedinfo.html:467]: Moved the JavaScript content to Main.js

File: seedinfo/seedinfo.html [hunk 4]
```diff
@@ -9,159 +8,7 @@
   <meta name="description" content="Displays information about how a seed was entered when creating a Minecraft World">
   <meta name="author" content="Panda4994">
 
-  <style>
```
Comment (Awe23123) [inline seedinfo/seedinfo.html:12]: Moved css content to Main.css

**Comment by Panda4994** [comment]
Heyho :-)
Thanks for the improvements!
I wasn't expecting that anyone would review my code :blush:

**Pull request merged**